### PR TITLE
feat: Aspose.PDF for .NET 23.9

### PR DESCRIPTION
**Pull request by alevsh**

### CODE REVIEW

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+|PDFNET-55157|Adding image with ImageFilterType.Flate does not preserve transparency|Feature|
+|PDFNET-55270|Provide a way to remove a child annotation from a fillable field|Feature|
+|PDFNET-44218|PDF/A-1b Conversion: Bad performance on some files|Enhancement|
+|PDFNET-55385|Enhance CheckBoxField.AddOption method behavior|Enhancement|
+|PDFNET-55343|Huge file size after copying fillable fields from another document|Enhancement|
+|PDFNET-53213|PDF Text changing and getting mirrored when extracting text|Bug|
+|PDFNET-54761|TextFragmentAbsorber not finding text extracted|Bug|
+|PDFNET-55037|Adding images is very slow with Flate filter (REPSVC-1478)|Bug|
+|PDFNET-49287|PDF to TIFF - Need to speed up the conversion process|Bug|
+|PDFNET-55338|Investigation: Exception occurred while optimizing and saving PDF|Bug|
```
Comment (AlexBiriukov) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:35]: We should remove this internal task from the list

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+|PDFNET-42802|Exception when joining 2 pdf files|Bug|
+|PDFNET-40958|PDF to PDF/A - text is corrupted and resultant file is not PDF/A compliant|Bug|
+|PDFNET-55195|Regression: Page.AddStamp() adds stamp to all pages|Bug|
+|PDFNET-55301|PDF to TXT: Text missing in output|Bug|
+|PDFNET-54912|There is a phenomenon where lines appear above and below the watermark|Bug|
+|PDFNET-54459|ArgumentOutOfRangeException occurs on using RedactionAnnotation.Redact()|Bug|
+|PDFNET-54955|System.NullReferenceException on PDF to DOCX conversion|Bug|
+|PDFNET-46919|PDF to TIF (Various parts of the document are disappearing)|Bug|
+|PDFNET-53709|PDFs will not save to XPS or print|Bug|
+|PDFNET-53768|PDF to PDF/A1a: "Required key missing" for StructTreeRoot|Bug|
```
Comment (AlexBiriukov) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:82]: We should remove the reopened PDFNET-53768 issue from the list.

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+|PDFNET-55301|PDF to TXT: Text missing in output|Bug|
+|PDFNET-54912|There is a phenomenon where lines appear above and below the watermark|Bug|
+|PDFNET-54459|ArgumentOutOfRangeException occurs on using RedactionAnnotation.Redact()|Bug|
+|PDFNET-54955|System.NullReferenceException on PDF to DOCX conversion|Bug|
+|PDFNET-46919|PDF to TIF (Various parts of the document are disappearing)|Bug|
+|PDFNET-53709|PDFs will not save to XPS or print|Bug|
+|PDFNET-42007|PageIndex throws ArgumentException|Bug|
+|PDFNET-38754|PDF to PDFA2A/PDFA2B/PDFA3B: Converting PDF, generated from pptx using Aspose.Slides, to PDFA2A/PDFA2B/PDFA3B fails the compliance|Bug|
+|PDFNET-54065|[Aspose.PDF.Drawing] Squares instead of Japanese characters when converting PDF on Linux|Bug|
+|PDFNET-53610|The gradient is lost when converting PDF|Bug|
```
Comment (AlexBiriukov) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:84]: Task PDFNET-53610 is Reopened and should be removed from the list

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+|PDFNET-55334|System.NullReferenceException while accessing to the Field.Parent property in certain cases|Bug|
+|PDFNET-55350|Widget page index is reset to 0 after importing a fillable field|Bug|
+|PDFNET-51102|Parallel processing - code is not executing properly|Bug|
+|PDFNET-54927|Page.AddStamp() throws System.IndexOutOfRangeException|Bug|
+|PDFNET-50853|Aspose.PDF 21.10: After adding annotation there are no annotations in the resulting pdf file if open it in Google Chrome|Bug|
+|PDFNET-54333|When FlatteningAnnotations with PdfAnnotationEditor Throws System.IndexOutOfRangeException|Bug|
+|PDFNET-47851|PDF to TIFF - text is overlapped in output|Bug|
+|PDFNET-47306|Blank DOCX file generated|Bug|
+|PDFNET-50637|Redaction area is not correct|Bug|
+|PDFNET-50374|Aspose.Pdf 21.7: Cannot load Pcl on macOS|Bug|
```
Comment (AlexBiriukov) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:94]: Issue PDFNET-50374 resolved with Aspose.PDF.Drawing (not Aspose.PDF for .NET). It should be removed from the list.

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-55157|Adding image with ImageFilterType.Flate does not preserve transparency|Feature|
+|PDFNET-55270|Provide a way to remove a child annotation from a fillable field|Feature|
+|PDFNET-44218|PDF/A-1b Conversion: Bad performance on some files|Enhancement|
+|PDFNET-55385|Enhance CheckBoxField.AddOption method behavior|Enhancement|
+|PDFNET-55343|Huge file size after copying fillable fields from another document|Enhancement|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:28]:
```suggestion
|PDFNET-55343|Reduced file size after copying fillable fields from another document|Enhancement|
```

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+|PDFNET-45377|Enhance the behavior of Annotation.CreationDate|Enhancement|
+|PDFNET-50826|Time while subsetting the fonts is improved|Enhancement|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:30]:
```suggestion
|PDFNET-50826|Reduced time for subset of fonts|Enhancement|
```

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+|PDFNET-53213|PDF Text changing and getting mirrored when extracting text|Bug|
+|PDFNET-54761|TextFragmentAbsorber not finding text extracted|Bug|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:32]:
```suggestion
|PDFNET-54761|TextFragmentAbsorber doesn't find extracted text|Bug|
```

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+|PDFNET-55037|Adding images is very slow with Flate filter (REPSVC-1478)|Bug|
+|PDFNET-49287|PDF to TIFF - Need to speed up the conversion process|Bug|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:34]: Maybe it is enhancement?

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+|PDFNET-45331|The output isn't PDF/A_2A compliant|Bug|
+|PDFNET-43418|Conversion to pdfa1b gives incompliant output|Bug|
+|PDFNET-44952|TIF image insertion to PDF document throws an exception|Bug|
+|PDFNET-46798|PDF to PDF/A - conversion corrupts content|Bug|
+|PDFNET-45392|Output intent related issue after pdf->pdf/a conversion|Bug|
+|PDFNET-52630|Issue in PageTreeNode.GetPage|Bug|
+|PDFNET-55159|XslFoToPdfConverterTests.Testimages - transparency of GIF images|Bug|
+|PDFNET-55300|Adding bookmarks using Aspose.Pdf is very very slow for large PDF files|Bug|
+|PDFNET-45870|PDF to PDF/A - Parameter not valid Exception occurs|Bug|
+|PDFNET-47854|AddPageBreak Method is not working correctly|Bug|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:46]:
```suggestion
|PDFNET-47854|AddPageBreak Method does not working correctly|Bug|
```

File: net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+|PDFNET-54485|Exception occurred while optimizing and saving PDF|Bug|
+|PDFNET-48657|Incorrect Cell borders with Colspan and wide tables|Bug|
+|PDFNET-42685|Document.BindXML() throws Typer Cell Does Not Exist Exception When TextSegment is empty|Bug|
+|PDFNET-44963|PDF To SVG - System.ArgumentException Exception occurred|Bug|
+|PDFNET-43118|System.Exception occurred at Document.Save() method resulting 0KB file|Bug|
+|PDFNET-53920|Chinese/Japanese characters in GoToRemoteAction brakes the Link Annotation|Bug|
+|PDFNET-53744|Image Left Margin does not work|Bug|
+|PDFNET-54301|Image horizontaly aligned to the right is outside the page|Bug|
+|PDFNET-54555|Document does not allow to add text, throwing 'Invalid font name' Exception|Bug|
+|PDFNET-53938|Rectangles not being drawn on document|Bug|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md:58]:
```suggestion
|PDFNET-53938|Rectangles are not drawn in the document|Bug|
```

File: net/release-notes/release-notes-2023/_index.md [hunk 11]
```diff
@@ -4,7 +4,7 @@ linktitle: Release Notes - 2023
 type: docs
 weight: 9
 url: /net/release-notes-2023/
-lastmod: "2023-08-11"
+lastmod: "2023-09-07"
 description: This section list all-new features, enhancement, and fixed bugs by Aspose.PDF for .NET library in 2023.
 sitemap:
     changefreq: "weekly"
```
Comment (andruhovski) [inline net/release-notes/release-notes-2023/_index.md:10]:
```suggestion
    changefreq: "monthly"
```

**Review by andruhovski — APPROVED** [review]

**Pull request merged**